.travis.sh

```diff
@@ -40,6 +40,45 @@ case "$1" in
       sudo mount -t mqueue none /dev/mqueue
     fi
 
+    if [[ "$DOCKER_VERSION" =~ ^1\.6\..* ]]; then
+      # docker-engine 1.6.x packages don't seem to have the upstart job config,
+      # so write it ourselves
+      echo '
```

Comment:
using a here-doc here would mean you don't have to worry about escaping the single-quotes below